source/layouts/layout.erb

```diff
@@ -109,7 +109,7 @@ under the License.
               <% if api_reference %><a class="toc-h1 toc-link"><span class="sidebar-section">REST API</span></a><% end %>
               <% if platform_ui_alpha %><a class="toc-h1 toc-link"><span class="sidebar-section">App Components</span></a></li><li class="platform-ui-alpha"><% end %>
```

Comment:
Should we also do something about this `platform_ui_alpha` now that we removed the beta?

Reply:
Fixed!